pom.xml

```diff
@@ -227,8 +227,8 @@
           <debug>true</debug>
           <optimize>true</optimize>
           <showDeprecation>true</showDeprecation>
-          <source>1.3</source>
-          <target>1.2</target>
+          <source>1.5</source>
```

Comment:
Let's drop this to 1.4 which I think was the first release to include DOM. Our source doesn't use any 1.5 features yet.